usr/src/cmd/truss/expound.c

```diff
@@ -1695,8 +1712,9 @@ show_statfs32(private_t *pri)
 		    statfs.f_fpack);
 	}
 }
-#endif	/* _LP64 */
+#endif	/* _MULTI_DATAMODEL */
 
+#if defined(_MULTI_DATAMODEL)
```

Comment:
I guess probably not worth joining this with the prior one.

Reply:
I thought about it, and decided it seemed the style to block them off individually so I stuck with it.  I'd thought about splitting it off into an `expound32.c` but I'm not sure that made things better enough

Reply:
Makes sense, that's fine.